docs/analytics.rst

```diff
@@ -118,3 +118,37 @@ When a banner is dismissed:
         'data-banner-dismissal': '1',
         'event': 'in-page-interaction'
     });
+
+
+When doing a/b tests configure something like the following.
+
+.. code-block:: javascript
```

Comment:
These aren't working. The blank line after the `code-block` declaration is required.